src/command/FileCommands.cxx

```diff
@@ -127,12 +127,7 @@ gcc_pure
 static bool
 IsValidValue(const StringView s) noexcept
 {
-	for (const char ch : s) {
-		if ((unsigned char)ch < 0x20)
-			return false;
-	}
-
-	return true;
+	return std::any_of(s.begin(), s.end(), [](const auto &ch) { return ch >= 0x20; });
```

Comment:
This omits the `unsigned char` cast and thus breaks the test.

Reply:
Also, this check is inverted. This checks if there is at least one valid character; but it is really important that *all* characters are valid.